ukat/mapping/tests/test_t1.py

```diff
@@ -275,15 +284,23 @@ def test_real_data(self):
         t1_stats = arraystats.ArrayStats(mapper.t1_map).calculate()
         npt.assert_allclose([t1_stats['mean']['3D'], t1_stats['std']['3D'],
                              t1_stats['min']['3D'], t1_stats['max']['3D']],
-                            gold_standard_3p, rtol=1e-6, atol=5e-3)
+                            gold_standard_3p, rtol=1e-4, atol=5e-2)
 
         # Three parameter method for first slice only
         mapper = T1(magnitude[:, :, 0, :], ti, affine, parameters=3,
                     tss=0, tss_axis=None)
         t1_stats = arraystats.ArrayStats(mapper.t1_map).calculate()
         npt.assert_allclose([t1_stats['mean'], t1_stats['std'],
                              t1_stats['min'], t1_stats['max']],
-                            gold_standard_3p_single, rtol=1e-6, atol=5e-3)
+                            gold_standard_3p_single, rtol=1e-6, atol=5e-2)
+
+        # MOLLI corrections/data
+        mapper = T1(image_molli, ti_molli, affine_molli, parameters=3,
+                    molli=True)
+        t1_stats = arraystats.ArrayStats(mapper.t1_map).calculate()
+        npt.assert_allclose([t1_stats['mean']['3D'], t1_stats['std']['3D'],
+                             t1_stats['min']['3D'], t1_stats['max']['3D']],
+                            gold_standard_molli, rtol=1e-6, atol=5e-3)
```

Comment:
Maybe the atol should also be 5e-2 for consistency?

Reply:
My logic is put the tolerances as tight as possible to allow things to pass on all systems/python versions, that way we're more likely to spot an issue if something causes a change in the results.